qiling/debugger/gdb/gdb.py

```diff
@@ -364,7 +363,7 @@ def handle_p(subcmd):
                             reg_value = self.addr_to_str(reg_value, short = True)
                     
                     elif self.ql.archtype== QL_ARCH.ARM:
-                        if reg_index < 17:
+                        if reg_index < 26:
```

Comment:
26 ? it never work for arm ?

Reply:
Thanks for your fast response!

Only in the **handle_g** call does it support a max of 16 core registers. In the **handle_p** and **handle_P** methods it will request registers by offset, where 25 is the CPSR. The registers in between are the old FP registers, which have become obsolete. 

As you can see with the debug logging, this is visible if we write the CPSR register from GDB:
![image](https://user-images.githubusercontent.com/59823130/113833701-442ccd80-978a-11eb-9300-31ed423a768c.png)

Which shows up in the logging as:

![image](https://user-images.githubusercontent.com/59823130/113833588-25c6d200-978a-11eb-8d0b-228684ab7bae.png)

In the GDB serial protocol, the 19 is the hexadecimal representation, which is decimal 25.